scripts/partition_manager.py

```diff
@@ -258,18 +258,48 @@ def get_config(reqs, config):
         raise RuntimeError("Unable to find '%s' in any of: %s" % (config, config_file))
 
 
-def get_flash_size(reqs):
+def get_free_size(reqs):
     return get_config(reqs, "CONFIG_FLASH_SIZE")
 
 
-def get_pm_config(input_config):
-    reqs = dict()
-    load_reqs(reqs, input_config)
-    flash_size = get_flash_size(input_config)
-    solution, sub_partitions = resolve(reqs)
-    set_addresses(reqs, sub_partitions, solution, flash_size)
-    set_sub_partition_address_and_size(reqs, sub_partitions)
-    return reqs
+def get_pre_defined_start_size(pre_defined_config, flash_size):
```

Comment:
either rename this to start_and_size, or if possible, find a way to have the function(s) return one value.